salt/orch/kubernetes.sls

```diff
@@ -66,9 +66,15 @@ etcd_discovery_setup:
     - sls:
       - etcd-discovery
     - require:
-      - salt: ca_setup
```

Comment:
I think the `ca_setup` is still a requirement here: we would need the CA in order to have certificates for `etcdctl` for setting up the discovery `etcd`, right?